src/main/java/dk/sdu/se_f22/contentmodule/management/Data/Database.java

```diff
@@ -0,0 +1,30 @@
+package dk.sdu.se_f22.contentmodule.management.Data;
+
+import java.sql.*;
+
+public class Database {
+    private Connection connection = null;
+    static protected Database database = null;
+
+    private Database() throws SQLException {
+        //DriverManager.registerDriver(new org.postgresql.Driver());
+        //connection = DriverManager.getConnection("jdbc:postgresql://localhost:5432/postgres","postgres","password");
+        connection = dk.sdu.se_f22.sharedlibrary.db.DBConnection.getPooledConnection();
```

Comment:
Please do this, at each location that the connection is needed, and close it after it has been used.
You can use it like this:

```java
try (Connection connection = DBConnection.getPooledConnection()) {
    // connection can then be used here
} catch (/* some exception */) {
    // code here
}
```

This will ensure, that the connection is only used, when needed, and ensure that a connection is not kept open, when not used.

Reply:
ok babe mb will fix 😎👍